### DISCUSSION TITLE
How to install windows in UEFI

### DISCUSSION THREAD

**Discussion by kuil-lxd** [cobbler/cobbler#2757] [Q&A]
![Snipaste_2021-08-17_15-26-44](https://user-images.githubusercontent.com/36268408/129683642-f396c1c4-d403-4f23-89cf-d2254735fcfd.png)
I have encountered an error while installing windows using cobbler

error: kernel too old.
error: you need to load the kernel first

Centos 7.9
Cobbler 2.8.5
syslinux 6.03

grub.cfg :
menuentry windows2016 {
    linuxefi /images/windows2016/memdisk  ksdevice=bootif lang=  text raw iso kssendmac  ks=http://1.1.1.16/cblr/svc/op/ks/profile/windows2016
    initrdefi /images/windows2016/win2016pe_cobbler_amd64.iso
}

What can I do about it?

**Comment by SchoolGuy** [comment]
@kuil-lxd So we havn't added Windows Support until [v3.2.1](https://github.com/cobbler/cobbler/releases/tag/v3.2.1) officially. So I can't help you with that. What I can say is that what you are doing is not working because GRUB2 won't chainload the Windows Bootloaders. Thus the `grub.cfg` you are showing would never work.
